### ISSUE TITLE
Ansible module (and couple of questions)

### ISSUE THREAD

**Issue by rlex** [alexbers/mtprotoproxy#37]
Hello! I wrote a simple ansible role for easy mtprotoproxy: https://github.com/rlex/ansible-role-mtprotoproxy
Might be useful for those who deploy bunch of proxies.

Also, is it possible to either move config.py to separate location (ie config.py.template) or maybe document mtprotoproxy.py arguments (ie that you can use config path after it to load something different than config.py)? It's not quite obvious and editing config.py while in working tree is not great idea, especially if you fetch changes pretty often.

**Comment by seriyps** [comment]
Readme says "uvloop is available in ubuntu but not in debian" but `python3-uvloop` is added to Ubunty.yml

Which one is correct?

**Comment by rlex** [comment]
Yep, package doesn't exist in debian stable. Probably can switch to pip installation if uvloop greatly helps with performance.

**Comment by alexbers** [comment]
Hello, thank you for the ansible role. I added a section about the advanced usage in readme: https://github.com/alexbers/mtprotoproxy/commit/63b689e3bf61cd701a4cdfb4a670cc913f1906b8
